app/src/pages/movies/filter/Filter.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { FilterListComponent } from './components/layouts/FilterListComponent';
+import { FilterOption } from './components/FilterOption';
+
+let options = [
```

Comment:
Do you plan to add new options? Maybe `const `is better?

Reply:
No, it is fixed array, fixed)